### DISCUSSION TITLE
Next Virtual Beers topics

### DISCUSSION THREAD

**Discussion by gionn** [gionn/montacchiello-developers#3] [Ideas]
Proponi un argomento per la prossima Virtual Beer!

**Comment by mistrett** [comment]
~Jenkins - pipeline as code~ https://www.youtube.com/watch?v=1IWdxjilNJU

**Comment by dfaggioli** [comment]
- ~Rolling or not rolling, snapshots, transactional server (or desktop): in how many ways can one install and use the openSUSE distributions?~ https://www.youtube.com/watch?v=lOywJXN_1OE

**Comment by andychiare** [comment]
Autenticazione fai da te o di terze parti (es. social login)?

**Comment by mbarto** [comment]
Tattiche di sopravvivenza per remote workers, ovvero come rendere accettabile, se non piacevole, il nostro nuovo modo di lavorare e di vivere

**Comment by mistrett** [comment]
Progressive web app and trust web Activity: evoluzione di una web app.

**Comment by mistrett** [comment]
Node.js under the hood
